config/routes.rb

```diff
@@ -1,10 +1,8 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
-      resources :tasks, only: [:index, :show, :create, :update] do
-        member do
-          patch :update_completion
-        end
+      resources :tasks, only: [:index, :show, :create, :update, :destroy] do
```

Comment:
[NIT]

Can make this array of symbols a bit cleaner


```suggestion
      resources :tasks, only: %i[index show create update destroy] do
```